docs/source/_templates/navbar.html

```diff
@@ -0,0 +1,13 @@
+<nav class="bd-navbar navbar">
+  <!-- Center: Navigation items -->
+  <div class="navbar-center-group">
+    <ul class="nav navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link" href="{{ pathto('index') }}">About</a>
```

Comment:
I don't think the About link is needed, as it just redirects to the index (the python-cokkiecutter link in the top left achieves this).